packages/naming.cell.match/cell-match.test.js

```diff
@@ -0,0 +1,133 @@
+            parse full path to elem mod2   → dd.blocks/bb/__e/_m/bb__e_m_v.t → { cell: { layer: 'dd', block: 'bb', elem: 'e', mod: 'm', val: 'v', tech: 't' } }
+
+            rejects alien block            → dd.blocks/qq/bb.t               → { isMatch: false }
+            rejects alien block mod        → dd.blocks/qq/_m/bb_m.t          → { isMatch: false }
+            rejects alien block mod2       → dd.blocks/qq/_m/bb_m_v.t        → { isMatch: false }
+            rejects alien block elem       → dd.blocks/qq/__e/bb__e.t        → { isMatch: false }
+            rejects alien block elem mod   → dd.blocks/qq/__e/_m/bb__e_m.t   → { isMatch: false }
+            rejects alien block elem mod2  → dd.blocks/qq/__e/_m/bb__e_m_v.t → { isMatch: false }
+            rejects alien elem             → dd.blocks/bb/__f/bb__e.t        → { isMatch: false }
+            rejects alien elem mod         → dd.blocks/bb/__f/_m/bb__e_m.t   → { isMatch: false }
+            rejects alien elem mod2        → dd.blocks/bb/__f/_m/bb__e_m_v.t → { isMatch: false }
+            rejects alien mod              → dd.blocks/bb/_n/bb_m.t          → { isMatch: false }
+            rejects alien mod2             → dd.blocks/bb/_n/bb_m_v.t        → { isMatch: false }
+            rejects alien mod in elem      → dd.blocks/bb/__e/_n/bb__e_m.t   → { isMatch: false }
+            rejects alien mod2 in elem     → dd.blocks/bb/__e/_n/bb__e_m_v.t → { isMatch: false }
+
+            find & reject file elem        → dd.blocks/bb/__e/bb__e.t/x.y        → { cell: { layer: 'dd', block: 'bb', elem: 'e', tech: 't' }, isMatch: false, rest: '/x.y' }
+            find & reject file block mod2  → dd.blocks/bb/_m/bb_m_v.t/x.y        → { cell: { layer: 'dd', block: 'bb', mod: 'm', val: 'v', tech: 't' }, isMatch: false, rest: '/x.y' }
+            find & reject file elem mod2   → dd.blocks/bb/__e/_m/bb__e_m_v.t/x.y → { cell: { layer: 'dd', block: 'bb', elem: 'e', mod: 'm', val: 'v', tech: 't' }, isMatch: false, rest: '/x.y' }
+        `],
+
+        'flat / modern': [flatModernMatch, rawses`
+            reject invalid block           → .blocks → { cell: null, isMatch: false }
+            reject typical path for layer  → common.blocks → { cell: null, isMatch: false }
+            reject nested block path       → blocks/button → { cell: null, isMatch: false }
+            reject invalid block: _button  → button/_button → { cell: null, isMatch: false }
+            reject nested scheme mod       → button/_mod → { cell: null, isMatch: false }
+            reject invalid block           → button/button → { cell: null, isMatch: false }
+            match partial block path       → blocks → { cell: null, isMatch: true }
+            parse typical block path       → blocks/blocks.css → { cell: { layer: 'common', block: 'blocks', tech: 'css' } }
+            parse typical block in layer   → blocks/blocks@ios.css → { cell: { layer: 'ios', block: 'blocks', tech: 'css' } }
```

Comment:
Wow! So here is a modern scheme. Want to see it nested

Reply:
Something like:
```
bb/bb.t                # previously common.blocks/bb/bb.t
bb/bb@desktop.t        # previously desktop.blocks/bb/bb.t
bb/bb@touch.t          # previously touch.blocks/bb/bb.t
bb/_e/bb_e.t           # previously common.blocks/bb/_e/bb_e.t
bb/_e/bb_e@desktop.t   # previously desktop.blocks/bb/_e/bb_e.t
bb/__m/bb__m.t         # previously common.blocks/bb/__m/bb__m.t
bb/__m/bb__m@desktop.t # previously desktop.blocks/bb/__m/bb__m.t
```